code/src/test/kotlin/com/expediagroup/sdk/core/http/StatusTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.expediagroup.sdk.core.http
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class StatusTest {
+
+    @Test
+    fun `should return correct Status for valid codes`() {
+        assertEquals(Status.CONTINUE, Status.fromCode(100))
+        assertEquals(Status.OK, Status.fromCode(200))
+        assertEquals(Status.MOVED_PERMANENTLY, Status.fromCode(301))
+        assertEquals(Status.BAD_REQUEST, Status.fromCode(400))
+        assertEquals(Status.INTERNAL_SERVER_ERROR, Status.fromCode(500))
+        assertEquals(Status.THIS_IS_FINE, Status.fromCode(218))
```

Comment:
Do you want to add the rest?

Reply:
I'll add common ones, but it's too much to cover all of them. they're unlikely to be used by any API..

Reply:
Done!